src/mumble/LookConfig.cpp

```diff
@@ -105,6 +107,20 @@ LookConfig::LookConfig(Settings &st) : ConfigWidget(st) {
 	
 }
 
+void LookConfig::updateFontInfo(QLabel *label, const QFont &font) {
+	label->setText(QString::fromLatin1("%1, %2pt").arg(font.family()).arg(font.pointSize()));
+}
+
+void LookConfig::updateFontInfo() {
+	updateFontInfo(s);
```

Comment:
What's `s`? I think there should be a comment here explaining what this call is supposed to do